dev-requirements.txt

```diff
@@ -0,0 +1 @@
+pre-commit == 2.20.0
```

Comment:
don't we want to also add isort, flake and black?

Reply:
I've added these dependencies as well as `pytest` and `pytest-cov`